desktop/release-notes.md

```diff
@@ -24,12 +24,119 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
+- Fixed a bug that didn't restart Docker Desktop after applying an update.
+- Fixed a bug that caused the connection to Docker to be lost when the computer sleeps if a user is using virtualization.framework and restrictive firewall software.
+- Fixed a bug that caused Docker Desktop to run in the background even after a user had quit the application.  Fixes [https://github.com/docker/for-mac/issues/6440]
+- Disabled both Virtualization Framework and VirtioFS for users running macOS < 12.5
+
+#### For Windows
+
+- Fixed a bug where versions displayed during an update could be incorrect. Fixes [for-win/issues#12822](https://github.com/docker/for-win/issues/12822).
+
+### Security 
+
+#### For all platforms
+- Fix RCE via query parameters in the message-box route in the Electron client.
+- Fix RCE via extension description/changelog which could be abused by a malicious extension.
+
+#### For Windows
+- Fixed a bypass for the `--no-windows-containers` installation flag which was introduced in version 4.11. This flag allows administrators to disable the use of Windows containers.
+- Fixed the argument injection to the Docker Desktop installer which may result in local privilege escalation.
+
 ## Docker Desktop 4.11.1
 2022-08-05
 
 > Download Docker Desktop
 >
-> {% include desktop-install.html %}
+> [Windows](https://desktop.docker.com/win/main/amd64/84025/Docker%20Desktop%20Installer.exe) |
+> [Mac with Intel chip](https://desktop.docker.com/mac/main/amd64/84025/Docker.dmg) |
+> [Mac with Apple chip](https://desktop.docker.com/mac/main/arm64/84025/Docker.dmg) |
+> [Debian](https://desktop.docker.com/linux/main/amd64/84025/docker-desktop-4.11.0-amd64.deb) |
+> [RPM](https://desktop.docker.com/linux/main/amd64/84025/docker-desktop-4.11.0-x86_64.rpm) |
+> [Arch package](https://desktop.docker.com/linux/main/amd64/84025/docker-desktop-4.11.0-x86_64.pkg.tar.zst)
+
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+  <div class="panel panel-default">
+    <div class="panel-heading" role="tab" id="headingSeven">
+      <h5 class="panel-title">
+        <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseSeven" aria-expanded="true" aria-controls="collapseSeven">
+          Checksums
+          <i class="fa fa-chevron-down"></i>
+        </a>
+      </h5>
+    </div>
+    <div id="collapseSeven" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingSeven">
+      <div class="panel-body">
+      <li><b>Windows:</b> SHA-256 8af32948447ddab655455542f6a12c8d752642a2bd451e2a48f76398cfd872b0</li>
+      <li><b>Mac Intel:</b> SHA-256 b2f4ad8fea37dfb7d9147f169a9ceab71d7d0d12ff912057c60b58c0e91aed35</li>
+      <li><b>Mac Arm:</b> SHA-256 a7d84117bef83764cb9bf275cd01b8ba0c43f08dbfe4d4a7d4f05549cdd81f54</li>
```

Comment:
Are the checksums up to date? I only verified this one on build 85629 and I got a different SHA-256 sum. 🤔